### DISCUSSION TITLE
Thank you for your Hacktoberfest 2023 contributions!

### DISCUSSION THREAD

**Discussion by SergeyRoyt** [mondaycom/vibe#2846] [Announcements]
# Thank you for your Hacktoberfest 2023 contributions!

Dear Contributors,

We want to express our heartfelt gratitude for your exceptional contributions during [Hacktoberfest 2023](https://hacktoberfest.com/) to the [monday-ui-react-core](https://github.com/mondaycom/monday-ui-react-core) and [vibe-storybook-components](https://github.com/mondaycom/vibe-storybook-components) packages.

Your dedication and expertise have significantly improved our open-source projects. Your hard work and commitment to the open-source community are truly appreciated. We are proud to have you as part of our community.

### Numbers
Overall, this year 17 different contributors managed to solve 64 issues, which is super impressive and impactful!

<img width="1017" alt="image" src="https://github.com/mondaycom/vibe-storybook-components/assets/104433616/1f624a66-3dcd-4de9-8154-3533e970614f">
<img width="1036" alt="image" src="https://github.com/mondaycom/vibe-storybook-components/assets/104433616/c72c7766-b6ae-4040-8306-fc89822c240a">

### Contributors

Each and one of your GitHub tags are now listed on our Storybook welcome pages of [monday-ui-react-core](https://style.monday.com/?path=/docs/welcome--docs) and [vibe-storybook-components](https://mondaycom.github.io/vibe-storybook-components/?path=/docs/welcome--docs) as a token of our deep appreciation.

Special thanks to our top 3 contributors this year:  
🥇 @Kritik-J - 19 PRs   
🥈 @Dhoni77 - 17 PRs  
🥉 @balajik - 12 PRs  

And thanks to all the participants:
@SoumyadiptoPal, @viditagrawal56, @jes14, @pedaars, @evadrake89, @anirudhsudhir, @Franqsanz, @HarshitVashisht11, @nabinbhatt, @naorpeled, @SarthakD15, @sayyedarib, @Vijeth56, @Ryan-Biondo

Thank you all for making Hacktoberfest 2023 a success for us! We'll be looking forward to see you again!

Sincerely,  
Vibe team,  
[monday.com](https://monday.com)

**Comment by orrgottlieb** [comment]
This is amazing!!!

**Comment by Kritik-J** [comment]
@SergeyRoyt, Thank you for the invaluable support and guidance.

**Comment by SarthakD15** [comment]
Thank you so much @SergeyRoyt !!

**Comment by viditagrawal56** [comment]
Thanks @SergeyRoyt for your support 🙏

**Comment by pedaars** [comment]
Thanks @SergeyRoyt looking forward to the next hacktoberfest.

**Comment by HarshitVashisht11** [comment]
Thank you @SergeyRoyt for your support ❤️

**Comment by biplobsd** [comment]
https://biplobsd.me/blogs/view/contributing-to-open-source-my-hacktoberfest-experience.md

I wrote a blog about my hacktoberfest journey. I mentioned this project..
